pyhms/demes/single_pop_eas/de.py

```diff
@@ -0,0 +1,210 @@
+import numpy as np
+import scipy
+
+from ...core.individual import Individual
+from ...core.population import Population
+from .common import VariationalOperator, apply_bounds
+
+
+def get_randoms(population: Population) -> np.ndarray:
```

Comment:
A name could be more specific because we are referring to the specific step in DE rather than some general utility sampling method.